src/tlo/analysis/life_expectancy.py

```diff
@@ -0,0 +1,305 @@
+"""
+This file will read in the output files generated by model runs
+then employs a series of helper functions to
+generate life tables to estimate life expectancy for each run/draw
+
+function: produce_life_expectancy_estimates() will produce summary statistics
+or will output life expectancy at birth for M and F separately for every
+run/draw if argument(...median=False)
```

Comment:
is there a reason to favour 'median' for the summary of this measure across runs (c.f. `mean`)?